### ISSUE TITLE
Duplicate tab/results for file

### ISSUE THREAD

**Issue by myrddincat** [Microsoft/vscode#61743]
Issue Type: <b>Bug</b>

When searching (CTRL-SHIFT-F) within my project folder, some of the classes (specifically, one class in this particular instance) show up twice.

I searched for a string of characters, and I found one of the files listed twice in the results. I opened both copies to see what was going on, made changes in one, saved that file and closed it. The other tab that originally had the same file open in it was then empty (The tab still existed, still had the file's name in it, still opened the same location within Explorer, but the window showed no lines of code. Not even a number on the left to indicate that it was possible to write anything in it.). I closed that tab, at which point I was asked if I wanted to save. I clicked 'No,' thinking it would overwrite my file with nothing. The tab closed, but it reverted all of the changes I made in the previous tab.

Afterward, I opened a (different) file by searching for it within the project folder with CTRL-P. Two tabs opened for that file. They functioned exactly as the two tabs mentioned a moment ago did. I made changes in one, saved and closed it, and then noticed that the other tab was completely empty. I decided to close and save changes this time, and my file just had the changes I made in the previous tab.

I'm not sure exactly how to reproduce this, as it seems to happen sporatically. I'm using the latest version of Insiders (updated this morning and double checked just before writing all of this).

I have not tried disabling extensions, as I need the extensions I have installed in order to work with Apex and SFDX.

VS Code version: Code - Insiders 1.29.0-insider (8aef44b6426b37ccb22026f56f52f42d136f4a0a, 2018-10-24T10:07:06.221Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2712)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.92GB (19.13GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
monokai-charcoal-high-contrast|74t|1.0.4
bracket-pair-colorizer|Coe|1.0.60
xml|Dot|2.3.2
gitlens|eam|8.5.6
LogFileHighlighter|emi|2.4.2
vscode-great-icons|emm|2.1.44
vscode-commands|fab|1.2.3
asciidecorator|hel|0.2.0
easy-icons|jam|0.3.1
apex|Joh|0.0.2
visualforce|Joh|1.0.2
vscode-simple-icons|Lau|1.8.7
theme-monokai-pro-vscode|mon|1.1.8
sublime-keybindings|ms-|4.0.0
indent-rainbow|ode|7.2.4
material-icon-theme|PKi|3.6.0
salesforcedx-vscode-apex|sal|44.3.0
salesforcedx-vscode-apex-debugger|sal|44.3.0
salesforcedx-vscode-apex-replay-debugger|sal|44.3.0
salesforcedx-vscode-core|sal|44.3.0
salesforcedx-vscode-lightning|sal|44.3.0
salesforcedx-vscode-visualforce|sal|44.3.0
trailing-spaces|sha|0.2.11
sfdx-cli|vib|0.0.1

(15 theme extensions excluded)

</details>

**Comment by roblourens** [comment]
Next time you see this, can you try copying the files in the search viewlet (just right click > copy) so we can look at their paths to see whether one is different?

Does the path for this files have any special//unicode characters?

**Comment by myrddincat** [comment]
I checked that very carefully yesterday. The paths were identical.

**Comment by roblourens** [comment]
Does the path for this files have any special/unicode characters?

**Comment by myrddincat** [comment]
No, and I've worked on this file before with no issues like this. It only started happening yesterday.
I haven't had any issues with it today, so maybe it has already been fixed?
I update Insiders daily.

**Comment by roblourens** [comment]
I can't think of any relevant changes, but let me know if you see it again

**Comment by myrddincat** [comment]
Will do. Thank you for looking into it. :)

**Comment by myrddincat** [comment]
Alright, it's happening again this morning.
I've copied the search viewlet (pasting it below).
In the pasted version, the first and last entries are identical. 
In the actual search viewlet, they're both at the top of the list.


=====


C:\Source\Production\Salesforce\src\Zuora\main\default\classes\CreateRequestContractCaseValidator.cls
  1,14: public class CreateRequestContractCaseValidator implements IOpportunityValidator {
  7,9: 	public CreateRequestContractCaseValidator() {
  11,9: 	public CreateRequestContractCaseValidator(ISetErrors errorMessageSetter, IRepository repository) {

C:\Source\Production\Salesforce\src\Zuora\main\default\classes\OpportunityTriggerHandler.cls
  7,22: 		validators.add(new CreateRequestContractCaseValidator());

C:\Source\Production\Salesforce\src\Zuora\tests\default\classes\ZTestCreateRequestContractCaseValidator.cls
  2,20: private class ZTestCreateRequestContractCaseValidator {
  4,17: 	private static CreateRequestContractCaseValidator sut;
  24,13: 		sut = new CreateRequestContractCaseValidator(errorSpy, repository);
  118,41: 	public class MockRepository implements CreateRequestContractCaseValidator.IRepository {

C:\Source\Production\Salesforce\src\Zuora\main\default\classes\CreateRequestContractCaseValidator.cls
  1,14: public class CreateRequestContractCaseValidator implements IOpportunityValidator {
  7,9: 	public CreateRequestContractCaseValidator() {
  11,9: 	public CreateRequestContractCaseValidator(ISetErrors errorMessageSetter, IRepository repository) {

**Comment by roblourens** [comment]
Was that file open when you did the search?

**Comment by myrddincat** [comment]
I don't believe so, but I've opened and closed so many files since then that I can't be 100% sure. 
I suppose I'll have to wait until it happens again and let you know.

**Comment by DarkLite1** [comment]
Having the same issue, as stated in #62395 on the latest insiders release:
```
Version: 1.30.0-insider (user setup)
Commit: c348be666adf9650b1ae1d212f019e6e9f226b7d
Date: 2018-11-21T06:25:32.454Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Windows_NT x64 6.2.9200
```

1. Open a file with the command palette quick open:
![image](https://user-images.githubusercontent.com/10828570/48833526-d54aa780-ed7b-11e8-8bd7-5ccea118e2f6.png)
2. When the file is opened and modified two tabs become available with the same file name:
![image](https://user-images.githubusercontent.com/10828570/48833580-f7dcc080-ed7b-11e8-8c90-01f12e1bca75.png)

Issues:
- Expected only 1 tab, but got two
- Changes are marked with a white circle in the second tab, not in the first one where they were made
- Duplicate entries for the same file in the quick open of the command palette
